src/store/features/worksSlice.ts

```diff
@@ -1,6 +1,16 @@
 import { createSlice } from '@reduxjs/toolkit';
 
-const initialState = {};
+export interface Work{
+    id : string,
+    title : string,
+    description : string
+}
```

Comment:
we can create a separate file/folder for storing types.

Reply:
okay, so for now should I remove it?

Reply:
store it in a separate file may be?

Reply:
okay